crates/scroll/chainspec/src/lib.rs

```diff
@@ -1,41 +1,41 @@
 //! Scroll-Reth chain specs.
 
-#![doc(
-    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
-    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
-    issue_tracker_base_url = "https://github.com/paradigmxyz/reth/issues/"
-)]
+#![doc = include_str!("../../../../docs/hardforks.md")]
```

Comment:
These are crate specific docs so I think they should remain within the crate as is done in `reth-scroll-trie` crate seen here https://github.com/scroll-tech/reth/tree/scroll/crates/scroll/trie/assets

Reply:
I proposed that we add this to `hardforks/src/lib.rs` but it looks like you added it to `chainspec` instead.

Reply:
yes, my fault, changed the wrong file, fixed[a188c1b](https://github.com/scroll-tech/reth/pull/56/commits/a188c1b4cdbb97d4c928d789454a8ee9f6be525d)